mobile/android/app/build.gradle

```diff
@@ -49,6 +49,16 @@ android {
         versionName flutterVersionName
     }
 
+    flavorDimensions "default"
+
+    productFlavors {
+        demo {
+            dimension "default"
+            resValue "string", "app_name", "10101 (regtest)"
```

Comment:
nit: The test app for ios is named 10101 Test

Reply:
I find it more intuitive to use the bitcoin network, but not sure we want to rename iOS.